lib/cms/sanity/index.ts

```diff
@@ -0,0 +1,14 @@
+import { createClient, type ClientConfig } from "@sanity/client";
+
+const PROJECT_ID = "4wbnjof1"; // TODO: move to env
```

Comment:
Does this need to be done?

Reply:
Should be in env on latest commit, but its pretty static, didnt change when I migrated the project over to another org.